00-RELEASENOTES

```diff
@@ -1,16 +1,154 @@
+* Optimize bitcount command by SIMD by @chzhoo (#1741)
+* Improve hash table performance efficiency by embedding hash value in hash type entry by @zuiderkwast (#1579)
+
+### Cluster
+* Add CLUSTER FLUSHSLOT command by @wuranxx (#1384)
+* Trigger manual failover on SIGTERM / shutdown to cluster primary by @enjoy-binbin (#1091)
+* Allow replicas to become primaries without data by using CLUSTER REPLICATE NO ONE by @skolosov-snap (#1674)
+* Add cluster-manual-failover-timeout to configure the timeout for manual failover by @enjoy-binbin (#1690)
+* Add cluster bus port out of range error message for CLUSTER MEET command by @hwware (#1686)
+
+### Module
+* Add new module API flag to bypass command validation by @sungming2 (#1357)
+
+### Bug Fixes
+* Redact user data when a module crashes for not handling I/O errors enabled by @YueTang-Vanessa (#2274)
+* Prevent bad memory access when NOTOUCH client gets unblocked by @uriyage (#2347)
+* Fix large allocations crashing Valkey during active defrag by @Fusl (#2353)
+* Fix replicas claiming to still have slots after manual failover by @enjoy-binbin (#2301)
+* Change default values of valkey-cli to use valkey naming, and added fallback to old values by @avifenesh (#2334)
+* Generate a new shard_id when the replica executes CLUSTER RESET SOFT by @enjoy-binbin (#2283)
+* Fix memory leak when shrinking a hashtable without entries by @yzc-yzc (#2288)
+* Fix missing response when AUTH returns an error inside a transaction by @enjoy-binbin (#2287)
+* Fix MEMORY USAGE to consider embedded keys by @yulazariy (#2290)
+* Fix use-after-free when active expiration triggers hashtable to shrink by @gusakovy (#2257)
+* Fix CLUSTER SLOTS/NODES showing wrong port after updating port/tls-port by @enjoy-binbin (#2186)
+* CLIENT UNBLOCK should't be able to unpause paused clients by @enjoy-binbin (#2117)
+* Fix replica can't finish failover when config epoch is outdated by @enjoy-binbin (#2178)
+* CVE-2025-32023 prevent out-of-bounds write during hyperloglog operations (#2146)
+* CVE-2025-48367 retry accept on transient errors (#2315)
+* Allow dynamic modification of io-threads num by @ayush933 (#2033)
+* Correctly handle large cluster bus extensions which may have resulted in dropped cluster packets by @madolson (#2144)
+* Improve clarity of errors for GEO commands when member does not exist by @chx9 (#1943)
+* Detect SSL_new() returning NULL in outgoing connections by @zuiderkwast (#2140)
+* Only mark the client reprocessing flag when unblocked on keys by @ranshid (#2109)
+* Fix memory corruption in sharded pubsub unsubscribe by @uriyage (#2137)
+* Respect process umask when creating data files by @kronwerk (#1725)
+* Allow mixing quoted and unquoted inline args by @Fusl (#2098)
+* Disallow sending REPLY ON / OFF / SKIP inside a multi-exec transaction by @sarthakaggarwal97 (#1966)
+* Environment variable REDISCLI_AUTH to VALKEYCLI_AUTH conversion by @HiranmoyChowdhury (#1995)
```

Comment:
Not sure if it's a bug. VALKEYCLI_AUTH wasn't documented. I'd put it under behavior change.

Calling it a "conversion" sounds odd too. The name is just changed, though we still read the old variable too so it's not a breaking change.

Mention somewhere that it's about valkey-cli.

```suggestion
* Support environment variable VALKEYCLI_AUTH alongside REDISCLI_AUTH in valkey-cli by @HiranmoyChowdhury (#1995)
```

Reply:
Moved and rephrased.